axelor-sale/src/main/resources/domains/SaleOrderLine.xml

```diff
@@ -109,6 +109,9 @@
     <one-to-many name="saleOrderLineTreeList" title="Sale order line trees"
       ref="com.axelor.apps.sale.db.SaleOrderLineTree" mappedBy="saleOrderLine"/>
 
+    <integer name="invoicingModeSelect" selection="line.invoicing.mode.select"
+      title="Invoicing mode"/>
+
```

Comment:
This should not be in the sale-module. It should be in the override of the SaleOrderLine domain in business-project.